bgpd/bgpd.c

```diff
@@ -5308,6 +5308,12 @@ int peer_timers_connect_set(struct peer *peer, uint32_t connect)
 	if (!CHECK_FLAG(peer->sflags, PEER_STATUS_GROUP))
 		return 0;
 
+	if (peer->status != Established) {
```

Comment:
I assume peer-group status is never Established, right? Do we even need this check + stop/start events for a peer-group and not members?

Reply:
I have changed the logic now. Thanks for pointing out.